leaflet-treering.js

```diff
@@ -2459,6 +2478,18 @@ function ViewData(Lt) {
       });
     });
   },
+  /**
+   * copy text to clipboard
+   * @function enable
+   */
+  copyToClipboard = function(allData){
```

Comment:
@thorn573 can chime in, but I think we want to have global functions at the bottom of the file, and stick to the convention the app is using, so 
`function copyToClipboard(allData) {} `

Reply:
There's an example of global/helper functions with current convention under branch 'insert-break-point-fix'.

Reply:
I changed the format and moved it to the bottom of the file! is the insert-break-point branch going to be merged soon?

Reply:
It won't be merged soon as it's been deemed not a priority. I'm going to keep it around for whenever insert break point becomes a priority, but it'll just live in branch limbo for awhile.